<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowChart</class>
 <widget class="QWidget" name="ShowChart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>3</y>
     <width>727</width>
     <height>604</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="T1Button">
        <property name="text">
         <string>T1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T2Button">
        <property name="text">
         <string>T2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T3Button">
        <property name="text">
         <string>T3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T4Button">
        <property name="text">
         <string>T4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T5Button">
        <property name="text">
         <string>T5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T6Button">
        <property name="text">
         <string>T6</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T7Button">
        <property name="text">
         <string>T7</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T8Button">
        <property name="text">
         <string>T8</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="T9Button">
        <property name="text">
         <string>T9</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="previousPage">
        <property name="text">
         <string>上一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextPage">
        <property name="text">
         <string>下一页</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="displayButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="dynamicWaveButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>动态波形</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fulllScreenButton">
        <property name="text">
         <string>全部</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="promptLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>P波到时手动调整</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>T1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>T9</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="intputPWave"/>
      </item>
      <item>
       <widget class="QPushButton" name="saveModifiedPWave">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>650</width>
        <height>500</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>9</number>
      </property>
      <widget class="QWidget" name="page1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item row="1" column="0">
           <widget class="QLabel" name="t1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>T1</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t1Z">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="t1X">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="t1Y">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1">
           <widget class="QLabel" name="t2X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="t2Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="t2">
            <property name="text">
             <string>T2</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t2Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QLabel" name="t3Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="t3Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="t3X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="t3">
            <property name="text">
             <string>T3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="1">
           <widget class="QLabel" name="t4X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="t4Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t4Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="t4">
            <property name="text">
             <string>T4</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="2">
           <widget class="QLabel" name="t5Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="t5">
            <property name="text">
             <string>T5</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="t5X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="t5Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="1">
           <widget class="QLabel" name="t6Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t6Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="t6">
            <property name="text">
             <string>T6</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="t6X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="1">
           <widget class="QLabel" name="t7Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="t7X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="t7">
            <property name="text">
             <string>T7</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t7Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="1">
           <widget class="QLabel" name="t8X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="t8">
            <property name="text">
             <string>T8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="t8Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t8Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="1" column="0">
           <widget class="QLabel" name="t9">
            <property name="text">
             <string>T9</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="t9Y">
            <property name="text">
             <string>Y-北南</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="t9X">
            <property name="text">
             <string>X-东西</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="t9Z">
            <property name="text">
             <string>Z-垂直</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>6</pointsize>
        </font>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout_10">
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="6" column="0">
           <widget class="QLabel" name="tt7">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T7</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="tt1">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T1</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="tt4">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T4</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="tt3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T3</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="tt9">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T9</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="tt8">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T8</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="tt2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T2</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="tt6">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T6</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="tt5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>T5</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
